.github/workflows/release-pr.yaml

```diff
@@ -98,6 +98,23 @@ jobs:
           branch: "release-${{ env.NEWVERSION }}"
           base: "${{ env.TARGET_BRANCH }}"
           signoff: true
+          body: |
+            ## Release Preparation Checklist
+
+            Before merging this PR, please ensure the following items are completed:
+
+            ### Pre-Release Dependencies Check
+            - [ ] **Dependabot PRs**: All open [Dependabot PRs](https://github.com/open-policy-agent/gatekeeper/pulls?q=is%3Apr+author%3Aapp%2Fdependabot) have been reviewed and merged
+            - [ ] **OPA**: Updated to latest stable version from [OPA releases](https://github.com/open-policy-agent/opa/releases)
+            - [ ] **cert-controller**: Updated to latest stable version from [cert-controller releases](https://github.com/open-policy-agent/cert-controller/releases)
```

Comment:
so far we only cut a CC release when requested. we dont always update GK to a CC release. mostly just to the latest commit. do we want to always cut a release when we need to bump it for a Gatekeeper release or is a commit ok since CC doesnt change as often?

Reply:
updated to commit instead of release